src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Controls/XamlEvent_Leak_TextBox.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml.Controls
+{
+    public sealed partial class XamlEvent_Leak_TextBox : UserControl
+    {
+        public XamlEvent_Leak_TextBox()
+        {
+            this.InitializeComponent();
+        }
+
+		private void Button_Click(object sender, RoutedEventArgs e)
+		{
+			Console.WriteLine("Button_Click");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6250494)